crypto/encrypt_test.go

```diff
@@ -212,3 +218,31 @@ func TestCiphertextMarshalling(t *testing.T) {
 		}
 	}
 }
+
+func TestTwofishNewCipherAssumption(t *testing.T) {
+	// Generate key.
+	key, err := GenerateTwofishKey()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// Test that len(key) is 16, 24, or 32 as these are the only cases
+	// where twofish.NewCipher(key[:]) doesn't return an error.
+	len := len(key)
```

Comment:
try to avoid using builtin functions as variable names. This would actually prevent you from calling `len` later on. `keyLen` is a better name